### DISCUSSION TITLE
always shows: this takes longer, getting status

### DISCUSSION THREAD

**Discussion by petjiang** [Vinzent03/obsidian-git#825] [Problems]
the obsidian-git plugin works well. but always shows (this takes longer, getting status).., even when i do nothing

**Comment by swz0169-hue** [comment]
I have also met this problem, and it's not working on my android phone, keeps pop out "This takes longer: Getting status", though I have no problem using termux to pull and push repo on my phone, still haven't find out the solution on git plugin.
